docs/source/triggers/index.rst

```diff
@@ -304,9 +304,9 @@ Let's walk through the ``rule`` code line-by-line:
 - We print the list of users we just fetched, as a debug message.
 - We create a function ``isSender`` to identify the user we are looking for.
 - We get the user contract by applying our ``isSender`` function as a
-  `filter </daml/stdlib/Prelude.html#function-da-internal-prelude-filter-27394>`_
+  `filter <https://docs.daml.com/daml/stdlib/Prelude.html#function-da-internal-prelude-filter-41317>`_
```

Comment:
This is not a dead link for me, either on https://docs.daml.com/triggers/index.html or https://docs.daml.com/2.5.0/triggers/index.html. How did you get a 404 here?

Reply:
Ah I see the number at the end has changed. Still, I'd rather we didn't use absolute links.

Reply:
i wasn't sure if this way was working but i'll try again..

Reply:
So, I can't test these locally as it is generated code but I have updated 3 here, head, filter, exercise (bearing in mind only 2 were reported).

Reply:
Here is a full, online build of this commit (26f100704d):
https://garyverhaegen-da.github.io/daml-docs/2022-11-24-26f100704d/triggers/index.html

Reply:
So the links are dead because they are absolute links. Which means on the live website they'll work, but lose the current version, which is a shame.